grails-app/controllers/au/org/ala/spatial/portal/LogController.groovy

```diff
@@ -26,9 +28,13 @@ class LogController {
         String url = "${grailsApplication.config.layersService.url}/log/search"
         def headers = [:]
         headers.put("apiKey",grailsApplication.config.api_key)
+        headers.put("userId", authService.userId)
+        headers.put("Accept", "application/json")
         request.headerNames.each { name -> headers.put(name, request.getHeader(name)) }
+        def params = [:]
```

Comment:
You shouldn't redefine `params` - it is injected into all controller methods by Grails via the `request` object. By re-initialising it, you are potentially making the code confusing for other devs who might need to maintain this code. You should use another variable name, like `serviceParams`. It will also prevent you having to use the less commonly-used: `request.getParameter(name)` below.